GenderPayGap.WebUI/Controllers/ViewingController.cs

```diff
@@ -212,7 +212,7 @@ public IActionResult SuggestEmployerNameJs(string search)
         [HttpGet("download")]
         public IActionResult Download()
         {
-            return View("Download");
+            return View("Download/Download");

```

Comment:
Why do we need this change?

Reply:
Since now we have multiple files download related I added a directory and moved all of them there. So the path's changed